backend/src/test/admin.test.js

```diff
@@ -1,10 +1,200 @@
+const {app,postgres} = require('../main'); 
+const cleardb = `
+TRUNCATE items, bids, users, shipments, payments; 
+`;
+const addFunds = `
+    UPDATE users 
+    SET balance = (balance + $2)
+    WHERE id = $1
+    RETURNING *`;
+const adminkey = '2c6rtf7gy8bFehUVfui373undbjhvUGuhBiUEefpwid';
+beforeEach(async() => {
+	try{
+		const client = await postgres.connect(); 
+		await client.query(cleardb); 
+		client.release(); 
+		return null; 
+	} catch(err) {
+		return err; 
+	}
+});
 const request = require('supertest');
-const app = require('../main'); 
 describe('Choose Winner', () => {
 	it('should only choose winners for items awaiting raffle', async () => {
-	
+		//create user create item set bid :q
+		//
+		const userData = {
+			"first_name": "test",
+			"last_name": "User",
+			"email": "user@test.com",
+			"password": "qwerty",
+			"pic_url": "<profile_picture_url>",
+			"address_1": "123 Address Lane",
+			"address_2": "This should be optional",
+			"city": "Los Angeles",
+			"state": "CA",
+			"zip": "90024",
+			"phone": "1234567890"
+		};
+		const userData2 = {
+			"first_name": "test3",
+			"last_name": "User",
+			"email": "user3@test.com",
+			"password": "qwerty",
+			"pic_url": "<profile_picture_url>",
+			"address_1": "123 Address Lane",
+			"address_2": "This should be optional",
+			"city": "Los Angeles",
+			"state": "CA",
+			"zip": "90024",
+			"phone": "1234567890"
+		};
+		const signUpResponse = await request(app).post('/api/users/signup')
+			.send(userData)
+			.set('Accept', 'applications/json')
+			.expect(200); 
+		const authenticating = await request(app).post('/api/users/login')
+			.send({"email":"user@test.com","password":"qwerty"})
+			.set('Accept', 'applications/json')
+			.expect(200); 
```

Comment:
Signup response will have the auth token

Reply:
Fixed this